examples/server.c

```diff
@@ -108,7 +108,9 @@ int main(int argc, char** argv) {
     signal(SIGINT, stopHandler); /* catches ctrl-c */
 
     UA_ServerNetworkLayer nl = UA_ServerNetworkLayerTCP(UA_ConnectionConfig_standard, 16664);
-    UA_ServerConfig config = UA_ServerConfig_standard;
+    UA_ServerConfig config;
+    UA_ServerConfig_standard_new(&config);
```

Comment:
Imho we can make UA_ServerConfig_standard_new return a malloced struct.
- It's consistent with how we use the _new postfix throughout the library.
- The config can be accessed from multiple threads. That is impossible when it "lives" somewhere on the stack.

I didn't want to pile too many features in this PR.
But since UA_ServerConfig_standard_new is newly created anyway, we can do it here.